tools/Signing.Dependencies/Signing.Dependencies.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Yes. There is a step in the Prepare.Targets that restores all projects under tools.